pkg/nsx/services/builder.go

```diff
@@ -639,7 +722,42 @@ func (service *SecurityPolicyService) buildRuleDstGroup(obj *v1alpha1.SecurityPo
 	return &ruleDstGroup, ruleDstGroupPath, err
 }
 
-func (service *SecurityPolicyService) buildPeerTags(obj *v1alpha1.SecurityPolicy, peers *[]v1alpha1.SecurityPolicyPeer, idx int) []model.Tag {
+func (service *SecurityPolicyService) buildRuleBasicInfo(obj *v1alpha1.SecurityPolicy,
+	rule *v1alpha1.SecurityPolicyRule, ruleIdx int, portIdx int, dupPortIdx int,
```

Comment:
Add comments to explain the function, especially the meaning of `portIdx` and `dupPortIdx`.

Reply:
ok